docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -591,7 +591,15 @@
    "source": [
     "## Parameter\n",
     "\n",
-    "Qcodes `Parameter` has the property that it is settable, gettable or both. Let us understand this with an example of a real instrument say oscilloscope. An oscilloscope contains a lot of settings such as trigger mode, trigger level, source etc. Most of these settings can be set to a particular value in the instrument say trigger mode can be set to 'edge' mode, trigger level to some floating number etc, hence these parameters are settable. Similarly, for many parameters, you may also be able to retrieve values hence they are gettable. In this example notebook, we have 'dac' instrument with 'ch1' and 'ch2' as `Parameter`s. Similarly, we have dmm instrument with 'v1' and 'v1' as `Parameters`. Apart from this obvious use of `Parameter` as the standard parameter of the instrument it may also be used as variable, i.e. storing/retrieving a value, or be subclassed for more complex uses."
+    "Qcodes `Parameter` has the property that it is settable, gettable or both. Let us understand this with an example of a real instrument say oscilloscope. An oscilloscope contains a lot of settings such as trigger mode, trigger level, source etc. Most of these settings can be set to a particular value in the instrument say trigger mode can be set to 'edge' mode, trigger level to some floating number etc, hence these parameters are settable. Similarly, for many parameters, you may also be able to retrieve values hence they are gettable. In this example notebook, we have 'dac' instrument with 'ch1' and 'ch2' as `Parameter`s. Similarly, we have dmm instrument with 'v1' and 'v1' as `Parameters`. Apart from this obvious use of `Parameter` as the standard parameter of the instrument it may also be used as variable, i.e. storing/retrieving a value, or be subclassed for more complex uses.\n",
```

Comment:
The dmm should have `v1` and `v2` as parameters (it accidentally says `v1` twice)

Reply:
Thanks! Corrected it.